src/support/LookupSummaryAndPolicies.tsx

```diff
@@ -34,6 +37,31 @@ export const LookupSummaryAndPolicies = (props: ResourceTypeSummaryProps) => {
     supportResources
   );
 
+  const supportSummaryProps: ResourceTypeSummaryProps = React.useMemo(() => {
```

Comment:
This was tricky: the `loadSupportSummaryFn` is built into the `ResourceTypeSummaryProps`, and request to load the resource (in this case workspace) are handled there, so I couldn't just define a single `loadSupportSummaryFn` in the `SupportResourceType`.  I'm not super thrilled about this solution, but it seems to work.  Am open to better ideas.